test/config/integration/server_xds.lds.udpa.list_collection.yaml

```diff
@@ -0,0 +1,26 @@
+version: "0"
+resource:
+  "@type": type.googleapis.com/envoy.config.listener.v3.ListenerCollection
+  entries:
+  - inline_entry:
+      name: listener_0
+      version: "0"
+      resource:
+        "@type": type.googleapis.com/envoy.config.listener.v3.Listener
+        name: listener_0
+        address:
+          socket_address:
+            address: {{ ntop_ip_loopback_address }}
+            port_value: 0
+        filter_chains:
+        - filters:
+          - name: http
+            typed_config:
+              "@type": type.googleapis.com/envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager
```

Comment:
v3? Not sure where this got copied from but should probably be updated there also.